librariejs.csproj

```diff
@@ -1,7 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Description>Project for nuget pack</Description>
+    <Description>Reusable lightweight library component based on React.js</Description>
     <Authors>Autodesk</Authors>
+    <Copyright>Autodesk 2019</Copyright>
+    <PackageProjectUrl>https://github.com/DynamoDS/librarie.js</PackageProjectUrl>
```

Comment:
@jesusHCG this should be MIT license - see the repo LICENSE file.

Reply:
it´s already added.